src/styles/pages/_live.scss

```diff
@@ -207,7 +240,9 @@ web-tabs.w-event-tabs {
   }
 
   .w-event-schedule__speaker {
-    width: #{$WIDTH_SPEAKER};
+    // width: #{$WIDTH_SPEAKER};
```

Comment:
remove?

Reply:
We can also remove `$WIDTH_SPEAKER` generally since both speaker and text is now full-width.

Reply:
oh ok